client/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.html

```diff
@@ -0,0 +1,84 @@
+            </div>
+            <div class="row">
+                <span class="col-sm-3 oui-tile__term"
+                    data-translate="xdsl_access_comfort_exchange_rma_equipment_reference"></span>
+                <span class="oui-tile__description" data-ng-bind="rma.equipmentReference"></span>
+            </div>
+        </div>
+    </div>
+
+    <div data-ng-if="!$ctrl.isRMAOpened" class="mb-3">
+        <div data-ng-if="!$ctrl.exchange.isSuccess">
+            <div class="mb-2" data-translate="xdsl_access_comfort_exchange_confirm_title"></div>
+            <div class="mb-2" data-translate="xdsl_access_comfort_exchange_confirm_subtitle"></div>
+
+            <oui-button variant="primary" data-ng-disabled="$ctrl.isAvailable"
+                data-ng-click="$ctrl.comfortExchange()">
+                <span data-translate="xdsl_modem_exchange"></span>
+            </oui-button>
+        </div>
+
+        <div data-ng-if="$ctrl.exchange.isSuccess">
+            <h3 data-translate="xdsl_access_comfort_exchange_bc_title"></h3>
+            <div class="row">
+                <span class="col-sm-3 oui-tile__term" data-translate="xdsl_access_comfort_exchange_bc_number"></span>
+                <span class="oui-tile__description" data-ng-bind="$ctrl.exchange.order.orderId"></span>
+            </div>
+            <div class="row">
+                <span class="col-sm-3 oui-tile__term" data-translate="xdsl_access_comfort_exchange_bc_date"></span>
+                <span class="oui-tile__description"
+                    data-ng-bind="($ctrl.exchange.order.date | date: 'dd/MM/yyyy HH:mm:ss' )"></span>
+            </div>
+            <div class="row">
+                <span class="col-sm-3 oui-tile__term"
+                    data-translate="xdsl_access_comfort_exchange_price_without_tax"></span>
+                <span class="oui-tile__description" data-ng-bind="$ctrl.exchange.order.priceWithoutTax.text"></span>
+            </div>
+            <div class="row">
+                <span class="col-sm-3 oui-tile__term" data-translate="xdsl_access_comfort_exchange_tax"></span>
+                <span class="oui-tile__description" data-ng-bind="$ctrl.exchange.order.tax.text"></span>
+            </div>
+            <div class="row">
+                <span class="col-sm-3 oui-tile__term"
+                    data-translate="xdsl_access_comfort_exchange_price_with_tax"></span>
+                <span class="oui-tile__description" data-ng-bind="$ctrl.exchange.order.priceWithTax.text"></span>
+            </div>
+
+            <div class="row">
+                <a target="_blank" class="oui-button oui-button_primary" rel="noopener"
+                    data-ng-href="{{:: $ctrl.exchange.order.url }}">
+                    <span data-translate="xdsl_access_comfort_exchange_bc_link"></span>
+                    <span class="oui-icon oui-icon-external_link" aria-hidden="true"></span>
```

Comment:
```suggestion
                    <span class="oui-icon oui-icon-external_link" aria-hidden="true"></span>
                    <span class="sr-only">Will open a new tab</span>
```
Otherwise screenreaders may not say it :)